velox/exec/tests/WindowTest.cpp

```diff
@@ -141,6 +141,40 @@ TEST_F(WindowTest, rowBasedStreamingWindowOOM) {
   testWindowBuild(false);
 }
 
+DEBUG_ONLY_TEST_F(WindowTest, aggWindowResultMismatch) {
```

Comment:
Why is this a debug-only test?

Reply:
@kagamiori Because we use the test value in this suite https://github.com/facebookincubator/velox/pull/10979/files#diff-9f41a5e1a8da742fc53f54ba85588678cdb81ffb05b38eefebe21ad5138def8eR162.